config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml

```diff
@@ -34,6 +35,7 @@ dashboards:
 - name: sig-cluster-lifecycle-cluster-api-1.8
 - name: sig-cluster-lifecycle-cluster-api-1.9
 - name: sig-cluster-lifecycle-cluster-api-1.10
+- name: sig-cluster-lifecycle-cluster-api-1.11
```

Comment:
capi is the only project with versioned dashboards. i was thinking that we should probably be fair to the rest of the subprojects and move all capi jobs to a single dashboard.

@fabriziopandini @sbueringer

Reply:
i think recently the capi operator requested the same version spread and i explained that's not ideal and noisy.

Reply:
What is the problem with having multiple dashboards? For us it's quite useful to have one dashboard per branch compared to all jobs for 4 branches in a single dashboard

Reply:
Sounds good, thx! Let's see were that discussion leads. Got the point about treating core CAPI differently then other SIG-CL sub-projects